test/unit/api/python/CMakeLists.txt

```diff
@@ -17,21 +17,22 @@
 check_python_module("pytest")
 
 # Add Python bindings API tests.
-macro(cvc5_add_python_api_test name filename)
+macro(cvc5_add_python_api_unit_test name filename)
 # We create test target 'python/unit/api/myapitest' and run it with
 # 'ctest -R "python/unit/api/myapitest"'.
-add_test (NAME python/unit/api/${name}
+add_test (NAME ${name}
```

Comment:
Can you remove the prefix from the Java tests as well for consistency?

Reply:
We decided offline to keep the prefix and use a prefix for all unit tests.